### PR TITLE
move engine for #47

### PR DESCRIPTION
**Pull request by starsplatter**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   mount Blacklight::Oembed::Engine, at: 'oembed'
   mount Riiif::Engine => '/images', as: 'riiif'
   root to: 'spotlight/exhibits#index'
-  mount Spotlight::Engine, at: 'spotlight'
+  mount Spotlight::Engine, at: '/'
```
Comment (cappadona) [inline config/routes.rb]: @starsplatter No ill effects from mounting both Blacklight and Spotlight at `/`?

Comment (starsplatter) [inline config/routes.rb]: I have not seen any yet:

http://exhibits.library.cornell.edu/jenn-test/catalog/3-1

This is what I was looking at from Stanford:

https://github.com/sul-dlss/exhibits/blob/master/config/routes.rb

Comment (cappadona) [inline config/routes.rb]: Thanks...I wonder if the [order matters](https://github.com/sul-dlss/exhibits/commit/4d0f6eeb56ce3e636571cf66e8f3eee952a1d8e4#diff-21497849d8f00507c9c8dcaf6288b136) but glad to hear it's running smoothly so far.

Comment (cappadona) [inline config/routes.rb]: #64 could be a side-effect of this...

**Pull request merged**